miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -813,12 +814,13 @@ public ModelAndView editPropagateSamples(@PathVariable String sampleIds, ModelMa
 
         if (isDetailedSampleEnabled()) {
           LibraryAdditionalInfoDto lai = new LibraryAdditionalInfoDto();
+          lai.setNonStandardAlias(((SampleAdditionalInfo) sample).hasNonStandardAlias());
           library.setLibraryAdditionalInfo(lai);
         }
-        libraries.add(library);
+        libraryDtos.add(library);
       }
       model.put("title", "Bulk Create Libraries");
-      model.put("librariesJSON", libraries);
+      model.put("librariesJSON", mapper.writeValueAsString(libraryDtos));
```

Comment:
I'll make a separate PR for this once I've finished refactoring TagBarcode -> Index